openwisp_users/accounts/templates/account/email/base_message.html

```diff
@@ -0,0 +1,23 @@
+{% extends "openwisp_utils/email_template.html" %}
+{% load i18n %}
+{% block body %}
+  <div class="container">
+    <div class="box">
+      {% block mail_header %}
+      <div class="logo-container">
+        <img src="https://raw.githubusercontent.com/openwisp/openwisp-utils/issues/259-email-template/openwisp_utils/static/openwisp-utils/images/openwisp-logo.png" alt="Logo" id="logo" class="logo">
```

Comment:
why are you redefining this? This defeats the purpose of the reusable template, if we configure a different logo, the configuration will have no effect because of this.